config_examples/societatis.json

```diff
@@ -0,0 +1,317 @@
+	},
+
+	"daemon": {
+		"host": "127.0.0.1",
+		"port": 7295
+	},
+
+	"wallet": {
+		"host": "127.0.0.1",
+		"port": 7296
+	},
+
+	"redis": {
+		"host": "127.0.0.1",
+		"port": 6379,
+		"auth": null,
+		"db": 0,
+		"cleanupInterval": 15
+	},
+
+	"notifications": {
+		"emailTemplate": "email_templates/default.txt",
+		"emailSubject": {
+			"emailAdded": "Your email was registered",
+			"workerConnected": "Worker %WORKER_NAME% connected",
+			"workerTimeout": "Worker %WORKER_NAME% stopped hashing",
+			"workerBanned": "Worker %WORKER_NAME% banned",
+			"blockFound": "Block %HEIGHT% found !",
+			"blockUnlocked": "Block %HEIGHT% unlocked !",
+			"blockOrphaned": "Block %HEIGHT% orphaned !",
+			"payment": "We sent you a payment !"
+		},
+		"emailMessage": {
+			"emailAdded": "Your email has been registered to receive pool notifications.",
+			"workerConnected": "Your worker %WORKER_NAME% for address %MINER% is now connected from ip %IP%.",
+			"workerTimeout": "Your worker %WORKER_NAME% for address %MINER% has stopped submitting hashes on %LAST_HASH%.",
+			"workerBanned": "Your worker %WORKER_NAME% for address %MINER% has been banned.",
+			"blockFound": "Block found at height %HEIGHT% by miner %MINER% on %TIME%. Waiting maturity.",
+			"blockUnlocked": "Block mined at height %HEIGHT% with %REWARD% and %EFFORT% effort on %TIME%.",
+			"blockOrphaned": "Block orphaned at height %HEIGHT% :(",
+			"payment": "A payment of %AMOUNT% has been sent to %ADDRESS% wallet."
+		},
+		"telegramMessage": {
+			"workerConnected": "Your worker _%WORKER_NAME%_ for address _%MINER%_ is now connected from ip _%IP%_.",
+			"workerTimeout": "Your worker _%WORKER_NAME%_ for address _%MINER%_ has stopped submitting hashes on _%LAST_HASH%_.",
+			"workerBanned": "Your worker _%WORKER_NAME%_ for address _%MINER%_ has been banned.",
+			"blockFound": "*Block found at height* _%HEIGHT%_ *by miner* _%MINER%_*! Waiting maturity.*",
+			"blockUnlocked": "*Block mined at height* _%HEIGHT%_ *with* _%REWARD%_ *and* _%EFFORT%_ *effort on* _%TIME%_*.*",
+			"blockOrphaned": "*Block orphaned at height* _%HEIGHT%_ *:(*",
+			"payment": "A payment of _%AMOUNT%_ has been sent."
+		}
+	},
+
+	"email": {
+		"enabled": false,
+		"fromAddress": "your@email.com",
+		"transport": "sendmail",
+		"sendmail": {
+			"path": "/usr/sbin/sendmail"
+		},
+		"smtp": {
+			"host": "smtp.example.com",
+			"port": 587,
+			"secure": false,
+			"auth": {
+				"user": "username",
+				"pass": "password"
+			},
+			"tls": {
+				"rejectUnauthorized": false
+			}
+		},
+		"mailgun": {
+			"key": "your-private-key",
+			"domain": "mg.yourdomain"
+		}
+	},
+
+	"telegram": {
+		"enabled": false,
+		"botName": "",
+		"token": "",
+		"channel": "",
+		"channelStats": {
+			"enabled": false,
+			"interval": 30
+		},
+		"botCommands": {
+			"stats": "/stats",
+			"enable": "/enable",
+			"disable": "/disable"
+		}
+	},
+
+	"monitoring": {
+		"daemon": {
+			"checkInterval": 60,
+			"rpcMethod": "getblockcount"
```

Comment:
"rpcMethod": "get_blockcount"

Reply:
Hi thanks. But no, we use a different notation than Monero. That fits.
`getblockcount`